fairlearn/post_processing/post_processing.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+class PostProcessing:
```

Comment:
Is the protected attribute going to change over different predictions? If not, could it be set once on the class and then reused?

Reply:
Yes and no! 
We will definitely not change the protected attribute itself after a round of fitting, that would make no sense. However, the protected_attribute variable itself is actually the data for the protected attribute, not a name of an attribute that's in X. I dislike the naming for that reason, and you mentioning this makes me think that we should rethink this argument name. @riedgar-ms  @MiroDudik  FYI

Reply:
Hmmmm…. we had been considering the attribute name as an argument to the disparity metrics. However in this case.... as you say, you only need the full list once.

Reply:
Leaving this unsolved for the broader discussion.